packages/nextjs/test/integration/test/server/tracingWithSentryAPI.js

```diff
@@ -0,0 +1,44 @@
+const assert = require('assert');
+
+const { sleep } = require('../utils/common');
+const { getAsync, interceptEventRequest, interceptTracingRequest } = require('../utils/server');
```

Comment:
```suggestion
const { getAsync, interceptTracingRequest } = require('../utils/server');
```

unused

Reply:
🍝 

Thanks. As an aside, it might be a good idea to run these tests through our normal eslint rules.